Makefile

```diff
@@ -53,7 +53,7 @@ ${VENV_ACTIVATE}: requirements.txt
 
 install-dependencies: ${VENV_ACTIVATE}
 
-docker/generated.mk: docker/generate_makefile.py fuzzers benchmarks ${VENV_ACTIVATE}
+docker/generated.mk: docker/generate_makefile.py docker/image_types.yaml fuzzers benchmarks ${VENV_ACTIVATE}
```

Comment:
Super smart, i missed this one, i was just trying yesterday on image_types.yaml changes.